mi-scheduler/base/cronjob.yaml

```diff
@@ -133,8 +131,86 @@ spec:
                       key: github-access-token
                 - name: SCHEDULE_KEBECHET_ANALYSIS
                   value: "1"
-                - name: SCHEDULE_GH_REPO_ANALYSIS
+                - name: THOTH_ADJUST_LOGGING
+                  value: 'thoth.mi-scheduler:INFO'
+          restartPolicy: OnFailure
+---
+kind: CronJob
+apiVersion: batch/v1beta1
+metadata:
+  name: mi-scheduler-kebechet-merge
+  labels:
+    app: mi-scheduler-kebechet-merge
+
+spec:
+  schedule: "00 22 * * *"
+  suspend: false
```

Comment:
can you suspend all cronjob in base 
and use the cronjob in individual overlays to suspend them.

please also include the cronjob mi-scheduler-kebechet-merge in all the overlays.